src/status_im/i18n.cljs

```diff
@@ -158,7 +158,7 @@
ount-generation-message :remove-network :no-messages :passphrase :recipient :members-title :new-group :phone-e164 :suggestions-requests :connected :rpc-url :settings :remove-from-group :specify-rpc-url :contacts-syncronized :transactions-sign-all :gas-limit :wallet-browse-photos :add-new-contact :no-statuses-discovered-body :add-json-file :delete :search-contacts :chats :transaction-sent :transaction :public-group-status :leave-chat :transactions-delete :mainnet-text :image-source-make-photo :chat :start-conversation :topic-format :add-new-network :save :enter-valid-public-key :faucet-error :phone-significant :all :confirmations-helper-text :search-for :sharing-copy-to-clipboard :your-wallets :phone-international :sync-in-progress :enter-password :enter-address :switch-users :send-transaction :confirmations :recover-access :incorrect-code :image-source-gallery :sync-synced :currency :status-pending :delete-contact :connecting-requires-login :no-hashtags-discovered-title :datetime-day :request-transaction :wallet-send :mute-notifications :invalid-phone :scan-qr :contact-s :unsigned-transaction-expired :status-sending :gas-used :transactions-filter-type :next :recent :confirmation-code :open-on-etherscan :share :status :from :your-current-location :wrong-password :search-chats :transactions-sign-later :in-contacts :transactions-sign :browsing-browse :sharing-share :type-a-message :type-a-command :usd-currency :existing-networks :node-unavailable :url :shake-your-phone :add-network :unknown-status-go-error :contacts-group-new-chat :and-you :wallets :clear-history :wallet-choose-from-contacts :signing-phrase-description :no-contacts :here-is-your-signing-phrase :soon :close-app-content :status-sent :status-prompt :testfairy-title :delete-contact-confirmation :datetime-today :add-a-status :web-view-error :notifications-title :error :transactions-sign-transaction :edit-contacts :more :cancel :no-statuses-found :can-not-add-yourself :transaction-description :add-to-contacts :available :open-map :paste-json-as-text :You :main-wallet :process-json :testnet-text :transactions :transactions-unsigned :members :intro-message1 :public-chat-user-count :eth :transactions-history :not-implemented :new-contact :datetime-second :search-results :status-failed :is-typing :sharing-copy-to-clipboard-address :recover :wallet-total-value :places-nearby :suggestions-commands :nonce :new-network :contact-already-added :datetime-minute :location :browsing-open-in-web-browser :delete-group-prompt :wallet-insufficient-funds :edit-profile :active-unknown :testfairy-message :search-tags :transaction-failed :public-key :error-processing-json :status-seen :transactions-filter-tokens :status-delivered :profile :wallet-choose-recipient :no-statuses-discovered :none :removed :empty-topic :no :transactions-filter-select-all :transactions-filter-title :message :here-is-your-passphrase :wallet-assets :image-source-title :current-network :left :edit-network-warning :to :data :cost-fee})
```

Comment:
Can we put these in a separate ns and/or make it so it is formatted correctly? I.e. <80ch per line